seclea_ai/seclea_ai.py

```diff
@@ -232,17 +238,40 @@ def upload_dataset(
                     )
             # upload all the datasets and transformations.
             for up_kwargs in upload_queue:
-                self._upload_dataset(**up_kwargs)  # TODO change
+                # self._upload_dataset(**up_kwargs)  # TODO change
```

Comment:
This is fine but you should also do the same and put the same entity into a send_q. Also I think it would be better to use entity rather than function as a differentiator in the dict.

Reply:
I have refactored, resolving this.